### PR TITLE
clarify Apache Maven vs individuals

### PR DESCRIPTION
**Pull request by hboutemy**
Clarification of Apache Maven in a few places.

I tried to clarify that individuals involved in this project will work as normal Apache Maven contributors

One remaining TODO to me is clarifying that funding is done for individuals (which ones?), not for the Apache project itself

### CODE REVIEW

**Comment by rmannibucau** [comment]
Guess something like "this project will found companies to let them involve Apache Maven contributors to work on Apache Maven" is closer to the reality since ultimately there is always a company hat between this and contributors?

File: README.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+# Support & Care for Apache Maven
+This repository is used to track the work of [Support & Care for Apache Maven](https://open-elements.com/support-care-maven/) as open and transparent as possible.
 
 <img width="1720" alt="logo" src="https://github.com/OpenElements/maven-support-care/assets/9443847/875e83bf-26e1-4488-a1fb-44e5de639f78">
 
-The **Support & Care for Maven** Initiative is a groundbreaking program to secure the future of the Maven project through sustainable support and care.
-The initiative aims to strengthen the development and maintenance of Maven through funding, thus ensuring its high quality and availability that the Java ecosystem needs.
+The **Support & Care for Maven** Initiative is a groundbreaking program to help secure the future of the [Apache Maven project](https://maven.apache.org/) through sustainable support and care.
+The initiative aims to strengthen the development and maintenance of Maven through funding of individuals involved in the project, thus ensuring its high quality and availability that the Java ecosystem needs.
```
Comment (hendrikebbers) [inline README.md:7]: @hboutemy I'm afraid I have to disagree with this change. Our goal is not to create funding for individuals. Our goal is to create / get funding for work done on the project. My suggestion:

```suggestion
The initiative aims to strengthen the development and maintenance of Maven through funding of needed contributions to the project, thus ensuring the high quality and availability of Apache Maven that the Java ecosystem needs.
```

File: README.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 
 ## Our Mission
 
-Maven is a critical tool in the Java ecosystem, used by millions of developers worldwide with >100 million downloads per year.
+Apache Maven is a critical tool in the Java ecosystem, used by millions of developers worldwide with >100 million downloads per year.
 Despite its importance, Maven is maintained primarily on a volunteer basis with no fixed financial support.
-Our mission is to create a sustainable source of funding to enable continuous improvements and professional support for Maven.
+Our mission is to create a sustainable source of funding for invidivuals to enable continuous improvements and professional support for Maven.
```
Comment (hendrikebbers) [inline README.md:13]: Suggest based on my previews argument:

```suggestion
Our mission is to create a sustainable source of funding for highly needed work around the Apache Maven project and direct contribution to the project. Our goal is to enable continuous improvements and professional support for Maven.
```

**Comment by hendrikebbers** [comment]
@hboutemy @rmannibucau I suggested some changes that focus on funding of project contribution. I dislike to speak about funding of individuals (or companies). While they will get the money it could be read in a negative way. Hope you like my suggestions.

**Comment by hendrikebbers** [comment]
> One remaining TODO to me is clarifying that funding is done for individuals (which ones?), not for the Apache project itself


Totally fine for me to add that information.

Suggestion to add the following to the end of the file:

## Relation to The Apache Software Foundation

Support&Care for Apache Maven is not related in any way to The Apache Software Foundation. This means that all resources that are managed by Support&Care for Apache Maven are not resources of The Apache Software Foundation or resources that are managed by The Apache Software Foundation.
On the other hand, the Support&Care for Apache Maven initiative would not be possible without the support of The Apache Software Foundation. Based on that, one of our main concerns is to never act against any decisions of The Apache Software Foundation. 

## Trademark

Apache™ and Apache Maven™ are trademarks of The Apache Software Foundation.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
-# Support & Care for Maven
-This repository is used to track the work of [Support & Care for Maven](https://open-elements.com/support-care-maven/) as open and transparent as possible.
+# Support & Care for Apache Maven
```
Comment (olamy) [inline README.md:1]: you will need to add few `&trade;`
see https://github.com/OpenElements/website/pull/89

**Review by olamy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hboutemy) [reply README.md:7]: @hendrikebbers I love it, thank you

**Review by olamy — CHANGES_REQUESTED** [review]
need some TM

**Comment by hboutemy** [comment]
IMHO, we can merge this PR and continue the improvements: step by step

**Comment by ascheman** [comment]
> IMHO, we can merge this PR and continue the improvements: step by step

LGTM, leaving the final decision to @hendrikebbers.

**Pull request merged**